### DISCUSSION TITLE
mismatch and gap next to each other

### DISCUSSION THREAD

**Discussion by divy-kangeyan** [snugel/cas-offinder#51] [Q&A]
<!DOCTYPE html>

Seq2 | RNA | CGCCAGCGTCGCAGCGACAGGTNNN | aGCCAGCt-CtCAGCGACAGcTGAG | chr1 | 91631879 | + | 3 | 1
-- | -- | -- | -- | -- | -- | -- | -- | --

Above is a row of the table from your readme. Why is the mismatch next to the gap is not counted. According to the DNA sequences there should have been 4 mismatches. I see the same in my data. So curious how you decide the mismatch count when the mismatch is next to a gap?

**Comment by DORISMO** [comment]
Hi, I cannot install the software. It suddenly quit. May I know how to fix it?
